modules/ai/module.php

```diff
@@ -837,11 +837,12 @@ public function ajax_ai_get_layout_prompt_enhancer( $data ) {
 		];
 	}
 
-	private function prepare_generate_layout_context() {
+	private function prepare_generate_layout_context( $data ) {
```

Comment:
You changed the signature, did you update all calls to this method?

Reply:
yes